app/Resources/views/base.mobile.html.twig

```diff
@@ -0,0 +1,109 @@
+                                  <img src="{{ asset('images/icons/profile.png') }}" />
+                                  <div >
+                                  {{ app.user.username }}                                                                
+                                     <a href="{{ path('fos_user_security_logout') }}">                                   
+                                        <img src="{{ asset('images/icons/logout.png') }}"  class="logout"/>
+                                     </a>
+                           
+                                </div>
+                            {% else %}
+                                <div>
+                                {{ 'anonymous' | trans }}
+                                 <a href="{{ path('fos_user_security_login') }}">
+                                    <img src="{{ asset('images/icons/login.png') }}" class="logout"/>
+                                 </a>
+                                </div>
+                            {% endif %}
+                        </span>
+                </div>
+                        
+                {% if not is_granted('ROLE_USER') %}
+                 <div class="menulist">                     
+                       <div class="menuelement" onclick='move("{{ path('fos_user_registration_register') }}");'>
+                         <img src="{{ asset('images/icons/registration.png') }}"/>
+                         <span> {{ 'registration' | trans }}</span>
+                     </div>
+                 </div>
+                 {% endif %}
+                 
+                <div class="menulist">
+                     <div class="menuelement" onclick='move("{{ path('homepage') }}");'>
+                         <img src="{{ asset('images/icons/home.png') }}"/>
+                         <span> {{ 'homepage' | trans }}</span>
+                     </div>
+                    {% if is_granted('ROLE_ADMIN') %}
+                    <div class="menuelement" onclick='move("{{ path('admin_dashboard') }}");'>
+                         <img src="{{ asset('images/icons/adminspace.png') }}"/>
+                         <span> {{ 'admin.dashboard' | trans }}</span> 
+                    </div>
+                    {% endif %} 
+                    {% if is_granted('ROLE_USER') %} 
+                         <div class="menuelement" onclick='move("{{ path('dashboard') }}");'>
+                             <img src="{{ asset('images/icons/myspace.png') }}"/>
+                             <span> {{ 'user.dashboard' | trans }}</span>
+                         </div>
+                    {% endif %}
+                    {% if is_granted('ROLE_USER') %}
+                          <div class="menuelement" onclick='move("{{ path('homepage') }}");'>
+                              <img src="{{ asset('images/icons/notification.png') }}"/>
+                              <span> {{ 'notifcation.all' | trans }}</span> 
```

Comment:
Typo here :)

Reply:
In fact is not really a typo error. I do not think we have created yet a view all notifications page. So I redirect to homepage for the moment ;-)

Reply:
I was talking about "'notifcation.all'" ;)